### PR TITLE
CF-1039 - Migrate archiving solution to Caspian Hadoop cluster

### PR DESCRIPTION
**Pull request by usnavi**
Adding new debug logging to help ascertain issues with identity.

### CODE REVIEW

File: tiamat/src/main/scala/com/rackspace/feeds/archives/Identity.scala [hunk 1]
```diff
@@ -93,15 +93,19 @@ class Identity( host : String, admin : String, apiKey : String, pw : String )  e
     val resp = client.execute(post)
 
     val output = Source.fromInputStream(resp.getEntity.getContent).mkString
-    val body = (new ObjectMapper).readTree(output)
+
+    logger.debug( "admin auth response: " + output )
```
Comment (ChandraAddala) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Identity.scala]: Wouldnt we be printing the token the logs if we do this? Thats got to violate some kinda security issue.

**Comment by usnavi** [comment]
@ChandraAddala  - good point.  I removed the debug statement which printed the content.  I still want to move the json parsing.

File: tiamat/src/main/scala/com/rackspace/feeds/archives/Identity.scala [hunk 2]
```diff
@@ -93,15 +93,19 @@ class Identity( host : String, admin : String, apiKey : String, pw : String )  e
     val resp = client.execute(post)
 
     val output = Source.fromInputStream(resp.getEntity.getContent).mkString
-    val body = (new ObjectMapper).readTree(output)
```
Comment (ChandraAddala) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Identity.scala]: If you are moviing this, can you move this in the other methods also?

Comment (usnavi) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Identity.scala]: @ChandraAddala - done.

**Comment by ChandraAddala** [comment]
@usnavi Looks good.

**Pull request merged**